common/js/src/popup-window/popup-opener.js

```diff
@@ -66,9 +67,11 @@ GSC.PopupOpener.createWindow = function(url, createWindowOptions, opt_data) {
 
   /** @preserveTry */
   try {
-    chrome.app.window.create(
-        url, createWindowOptions,
-        createWindowCallback.bind(null, createdWindowExtends));
+    if (GSC.Packaging.MODE == GSC.Packaging.Mode.APP) {
+      chrome.app.window.create(
+          url, createWindowOptions,
+          createWindowCallback.bind(null, createdWindowExtends));
+    }
```

Comment:
So this PR doesn't add the code for opening the popup in the extension mode? Was it forgotten, or is the plan to put it into a separate PR? (which would also be fine)

Reply:
yeah leaving that for a separate PR